warp10/src/main/java/io/warp10/hadoop/Warp10InputFormat.java

```diff
@@ -191,6 +192,7 @@ public class Warp10InputFormat extends InputFormat<Text, BytesWritable> {
   public static final String HTTP_HEADER_POSTBOUNDARY = "X-Warp10-Postboundary";
   public static final String HTTP_HEADER_TIMESTEP = "X-Warp10-Timestep";
   public static final String HTTP_HEADER_SHOW_ERRORS = "X-Warp10-ShowErrors";
+  public static final String HTTP_HEADER_ATTR_PREFIX = "X-Warp10-Attr.";
```

Comment:
boundary.post  matching header is X-Warp10-Postboundary, and this is a legal http header.

But attr.this.that  matching header will be X-Warp10-Attr.attr.this.that

Lots of software will filter out the header because of the dot (nginx for example).
This may cause troubles.